lib/charms/prometheus/v1/prometheus.py

```diff
@@ -76,7 +76,7 @@ def add_endpoint(self, address, port=80, rel_id=None):
         if address in targets:
             return
 
-        target = address if port == 80 else address + ":" + str(port)
+        target = address + ":" + str(port)
```

Comment:
may we use f-strings? `target = f"{address}:{port}"`

Reply:
Requires 3.6. But I agree good idea.

Reply:
As far as I remember, our charms have yet to be compatible with Python 3.5. 

If so we can use:

```python
target = "{}:{}".format(address, port)
```

Anyway it is a matter of taste. I prefer string interpolation over concatenation.

Reply:
Indeed this is a mater of taste. Here is my question to both of you. Should "matters of taste"  be blocking issues for code review ? Or should they just be comments (identified as such) left to the discretion of the code author ?

Reply:
+1 for f-strings but yes it doesn't really matter

String formatting of some sort, though, is typically preferred over concatenation as the language will occasionally give minor optimizations with formatting.

Reply:
I like string formatting too. But there are multiple places in this code where it needs to be done or it will be an inconsistent style. I will pick this up in a future commit. Will merge for now.